index.bs

```diff
@@ -986,191 +992,191 @@ The "Version(s)" column in the following table lists the version(s) of the boxes
+            <td>[[!HEIF]]</td>
             <td>&nbsp;</td>
             <td>Used when images in a group form a stereo pair</td>
         </tr>
         <tr>
             <td>pasp</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal pixel aspect ratio. If present, shall indicate a pixel aspect ratio of 1:1</td>
         </tr>
         <tr>
             <td>colr</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal information such as color primaries.</td>
         </tr>
         <tr>
             <td>auxC</td>
             <td>&nbsp;</td>
             <td>0</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>ipco</td>
             <td>Used to signal the type of an auxiliary image (e.g. alpha, depth).</td>
         </tr>
         <tr>
             <td>clap</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal cropping applied to an image</td>
         </tr>
         <tr>
             <td>irot</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>ipco</td>
             <td>Used to signal a rotation applied to an image</td>
         </tr>
         <tr>
             <td>imir</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>HEIF</td>
+            <td>[[!HEIF]]</td>
             <td>ipco</td>
             <td>Used to signal a mirroring applied to an image</td>
         </tr>
         <tr>
             <td>clli</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal HDR light level information for an image</td>
         </tr>
         <tr>
             <td>cclv</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal HDR color volume for an image</td>
         </tr>
         <tr>
             <td>mdcv</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>ISOBMFF</td>
+            <td>[[!ISOBMFF]]</td>
             <td>ipco</td>
             <td>Used to signal HDR mastering information for an image</td>
         </tr>
         <tr>
             <td>a1op</td>
             <td>&nbsp;</td>
             <td>-</td>
-            <td>AVIF</td>
+            <td>[=/AVIF=]</td>
             <td>ipco</td>
-            <td>Used to configure rendering of a multiple operating points image</td>
+            <td>Used to configure rendering of an image containing multiple operating points</td>
```

Comment:
Based on your comment above, and assuming "image" here means "AV1 image item" and not "AV1 image item data", should it rather be:

> Used to configure the operating point to render an image with when there are multiple choices

Reply:
Yeah, something like that would be better. Let me change it.